test/p2p/fast_sync/restart_peer.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/bash
+set -eu
+set -o pipefail
+
+###############################################################
+# for each peer:
+# 	kill peer
```

Comment:
It says kill peer here, and the name is restart_peer.sh, but I don't see any logic for killing peers in this file.  Misnomer?  Need refactor?